core/roboconf-dm/src/main/java/net/roboconf/dm/management/Manager.java

```diff
@@ -866,6 +871,21 @@ void deployRoot( ManagedApplication ma, Instance rootInstance ) throws TargetExc
 			Map<String,String> targetProperties = new HashMap<String,String>( target.getProperties());
 			targetProperties.putAll( rootInstance.data );
 
+			// Deploying VM into a VM (eg. Docker)
+			// TODO Other targets than Docker eligible ??
+			if(rootInstance.getParent() != null) {
```

Comment:
No, delete this.
It would have been more simple to rebase your repository to work on the script issue. :)